### DISCUSSION TITLE
Error starting using Aleph

### DISCUSSION THREAD

**Discussion by HValG** [alephjs/aleph.js#421] [Q&A] [answered]
Hello all,

I wanted to try out deno/Aleph but I encountered an issue as soon as I started :'( 

I installed latest deno and latest aleph, then for each aleph command i run I have this kind of error:

aleph dev
Check https://deno.land/x/aleph@v0.3.0-beta.19/cli.ts
error: Uncaught ReferenceError: Level is not defined
  #level: Level = Level.Info
                        ^
    at Logger.<instance_members_initializer> (https://deno.land/x/aleph@v0.3.0-beta.19/shared/log.ts:14:25)
    at new Logger (https://deno.land/x/aleph@v0.3.0-beta.19/shared/log.ts:13:7)
    at https://deno.land/x/aleph@v0.3.0-beta.19/shared/log.ts:101:16

I probably did something wrong ?

**Comment by ije** [comment, marked as answer]
this happen in deno 1.15, i will fix it asap!

**Comment by HValG** [comment]
Ho I see ! Tried with deno 1.14.3 and it's working.

Used deno upgrade --version 1.14.3 to downgrade.

Thank you :)

**Comment by ije** [comment]
upgrade to 1.15.1 resolves this too
